.github/workflows/pr-build-live-branch.yml

```diff
@@ -1,8 +1,7 @@
-name: Build PR Live Branch
+name: Build Live Branch
 on:
     pull_request:
         paths-ignore:
-            - '.github/**'
```

Comment:
I removed this for testing. I will revert this again once I have approval to merge.